pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -189,8 +189,16 @@ type OFBridge struct {
 	multipartReplyChs map[uint32]chan *openflow13.MultipartReply
 }
 
+func (b *OFBridge) CreateGroupTypeAll(id GroupIDType) Group {
```

Comment:
`CreateGroupTypeAll(id GroupIDType)` and  `CreateGroup(id GroupIDType)` confuses me. Why not `CreateGroupTypeAll(id GroupIDType)` and `CreateGroupTypeSelect(id GroupIDType)` or just `CreateGroup(id GroupIDType, , groupType ofctrl.GroupType)` ?

Reply:
Since `CreateGroup` is an existing function to use type "select" when creating OpenFlow group, which is called in AntreaProxy function. I don't want to change the existing code unrelated with this change, that's why I didn't rename it.  @hongliangl Would you share your point, do you think the existing function `CreateGroup` as `CreateGroupTypeSelect` is better?

Reply:
I think we can made another PR to rename `CreateGroup` to `CreateGroupTypeSelect` since we have another type of group.